bundles/org.openhab.binding.kostalinverter/src/main/resources/OH-INF/thing/Channels.xml

```diff
@@ -738,15 +738,15 @@
 	</channel-type>
 	<channel-type id="statistic-own-consumption-total">
 		<item-type>Number:Energy</item-type>
-		<label>Own Consumption Total</label>
+		<label>Total own consumption</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions